### DISCUSSION TITLE
Feature: Subscription Tab

### DISCUSSION THREAD

**Discussion by Enzoss100** [dailydotdev/daily#1175] [Feature Requests]
# Addition of a new Sidebar Option: Subscriptions
## Purpose: to easily access and edit subscriptions

**TLDR:** A Subscriptions Menu accessible from the sidebar, that looks and feels similar to Youtube Subscriptions

### Benefits:
- Easy access to all content from sources that the user is subscribed to
- Easy way to remove subscriptions if the user decides to not follow a certain source

### Feature Specifications:
- Accessible from the sidebar (hopefully under the `Feed` button)
- Ability to group subscriptions by topic (ex: `github`, `rust`, `database`)
- `Edit Subscriptions` button, which allows the user to remove subscriptions from their feed
- In it's own section, subscriptions are shown to the user with the most recent release at the top (can be sorted to look like a gallery or a timeline)

### Disclaimer
**This is my first time here**, and I just wanted to drop this suggestion because it was the first thing i looked for when I started subscribing to sources. I do not know if this was suggested yet.

**Comment by idoshamun** [comment]
Hi @Enzoss100,

Welcome! I guess it's similar to https://github.com/dailydotdev/daily/discussions/922 (which was opened before we introduced subscribe to source even)?

**Reply by Enzoss100** [reply]
Well in that case, I guess i just laid things out more and built up on it! Hope it helps, and great work with the project, I love what I am seeing so far
